network/p2p/gossip/mempool.go

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package gossip
+
+import (
+	"fmt"
+
+	"github.com/ava-labs/avalanchego/ids"
+
+	xmempool "github.com/ava-labs/avalanchego/vms/txs/mempool"
```

Comment:
We should avoid having a dependency on `vms`... I think one way to avoid this is to just remove the error type we're importing but if we really wanted to keep an error we should just define it here instead.

Reply:
ok, I'll "port" the error type into here.